src/main/java/org/gridsuite/shortcircuit/server/ShortCircuitController.java

```diff
@@ -85,7 +85,7 @@ private static List<UUID> getNonNullOtherNetworkUuids(List<UUID> otherNetworkUui
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The short circuit analysis result"),
         @ApiResponse(responseCode = "404", description = "Short circuit analysis result has not been found")})
     public ResponseEntity<ShortCircuitAnalysisResult> getResult(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid,
-                                                                @Parameter(description = "Full or summary result") @RequestParam(name = "full", required = false, defaultValue = "false") Boolean full) {
+                                                                @Parameter(description = "Full or summary result") @RequestParam(name = "full", required = false, defaultValue = "true") Boolean full) {
```

Comment:
it's never changed by the way, because the study-server nerver calls it with a full parameter value